lib/network/modules/Canvas.js

```diff
@@ -249,8 +250,7 @@ class Canvas {
     this.hammer.on('pinch',     (event) => {this.body.eventListeners.onPinch(event)});
 
     // TODO: neatly cleanup these handlers when re-creating the Canvas, IF these are done with hammer, event.stopPropagation will not work?
-    this.frame.canvas.addEventListener('mousewheel',     (event) => {this.body.eventListeners.onMouseWheel(event)});
-    this.frame.canvas.addEventListener('DOMMouseScroll', (event) => {this.body.eventListeners.onMouseWheel(event)});
```

Comment:
Why? I see absolutely no reason to do this. Just delete DOMMouseScroll and replace mousewheel by wheel.